app/src/js/components/Executions/overview.js

```diff
@@ -66,7 +66,7 @@ class ExecutionOverview extends React.Component {
               Execution Overview
             </h1>
             {lastUpdated(queriedAt)}
-            <Overview type='executions' inflight={false} />
+            <Overview type='executions' inflight={executions.list.inflight} />
```

Comment:
Awesome! Can we do something similar in the other places we're using the Overview component?

Reply:
Yup, will get those changes out!